src/tools/rbd/CMakeLists.txt

```diff
@@ -48,15 +48,22 @@ set(rbd_srcs
   action/Trash.cc
   action/Watch.cc)
 
+if(CURSES_FOUND)
```

Comment:
Nit: this could move inside `Perf.cc` since the `rbd perf image iostat` can function w/o curses.